### PR TITLE
Verify that Strings Are in Valid Email Format: inform about the MailAddress class from start

### PR DESCRIPTION
**Pull request by pkulikov**
- Moved the note about `System.Net.Mail.MailAddress` to the beginning of the topic. 
- Rephrased it such that it should be considered as the main alternative. Should it start with *We recommend...* to make even more stronger?
- Updated the first sentence to relate the whole topic to the set of regex examples.
 
Related to #5305

### CODE REVIEW

File: docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md [hunk 1]
```diff
@@ -22,7 +22,10 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # How to: Verify that Strings Are in Valid Email Format
-The following example uses a regular expression to verify that a string is in valid email format.  
+The following example is more complex than previous examples and uses a regular expression to verify that a string is in valid email format.
```
Comment (rpetrusha) [inline docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md]: This change is problematic. Previous here has no referent. The implicit referent is the TOC, but an extremely small number of people navigate here from the TOC or use the TOC. There's also no clear relationship in terms of complexity between examples. What is the case is that this example is complex. It's OK to phrase this as "The following example uses a complex regular expression..."

Comment (rpetrusha) [inline docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md]: Previous here has no referent. The implicit referent is the TOC, but very few people use it for navigation. The original sentence is succinct and to the point; it's best to leave it as is, or at most to change it to "The following example uses a complex regular expression..."

File: docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md [hunk 2]
```diff
@@ -22,7 +22,10 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # How to: Verify that Strings Are in Valid Email Format
-The following example uses a regular expression to verify that a string is in valid email format.  
+The following example is more complex than previous examples and uses a regular expression to verify that a string is in valid email format.  
+
+> [!NOTE]
+>  Use the <xref:System.Net.Mail.MailAddress?displayProperty=nameWithType> class to check if a string is in valid email address format. To do that, pass the email address string to the <xref:System.Net.Mail.MailAddress.%23ctor%28System.String%29?displayProperty=nameWithType> class constructor that throws a <xref:System.FormatException> if the string has unrecongnized format.
```
Comment (rpetrusha) [inline docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md]: constructor that throws --> constructor, which throws
has --> has an
unrecongnized --> unrecognized

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for submitting this PR, @pkulikov. I've left a number of comments.

**Comment by pkulikov** [comment]
@rpetrusha thank you for the review; I've addressed your feedback. Indeed, my editing of the first sentence was quite weak, I've reverted it to the original version.

**Review by rpetrusha — APPROVED** [review]
Thanks for making the additional set of changes, @pkulikov. I'll merge your PR now.

**Pull request merged**